CppSamples/ChilliConnectCppDemo/ChilliConnect.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include "HttpSystem.h"
+#include "json/json.h"
+
+#include <string>
+#include <memory>
+
+using std::string;
+using std::unique_ptr;
+using std::tuple;
+using std::vector;
+
+struct ChilliConnectResponse
+{
+	bool wasOk;
+	int errorCode;
+	string errorMessage;
+	string rawBody;
+};
+
+struct CreatePlayerResponse : ChilliConnectResponse
+{
+	string chilliConnectId;
+	string chilliConnectSecret;
+};
+
+struct LoginResponse : ChilliConnectResponse
+{
+	string accessToken;
+};
+
+struct AddScoreResponse : ChilliConnectResponse
+{
+	int globalRank;
+	int totalScores;
+};
+
+struct GetScoresResponse : ChilliConnectResponse
+{
+	vector<tuple<string, int, string>> scores;
```

Comment:
Personally I would have used another struct rather than a tuple as it gives access by name rather than by index